lib/internal/modules/cjs/loader.js

```diff
@@ -85,6 +85,7 @@ const manifest = getOptionValue('--experimental-policy') ?
   require('internal/process/policy').manifest :
   null;
 const { compileFunction } = internalBinding('contextify');
+const userConditions = (getOptionValue('--conditions') || '').split(',');
```

Comment:
wouldn't this always set a "" condition?

Reply:
Yes and it's causing a failing test, just pushing up a fix now.